pipelined/bevy_pbr2/src/render/mesh.rs

```diff
@@ -469,6 +475,9 @@ impl SpecializedPipeline for MeshPipeline {
             depth_write_enabled = true;
         }
 
+        #[cfg(feature = "webgl2")]
+        shader_defs.push(String::from("NO_ARRAY_TEXTURES_SUPPORT"));
```

Comment:
I think it is specifically no cube array texture support.

Reply:
textures of type `D2Array` seems to be an issue too, if I don't change them to `D2` I don't have shadows for directional lights

Reply:
Ah, ok. Well then I guess this is fine. :)